src/mono/wasm/runtime/debug.ts

```diff
@@ -363,27 +364,31 @@ export function mono_wasm_debugger_log(level: number, message_ptr: CharPtr): voi
         return;
     }
 
-    console.debug(`Debugger.Debug: ${message}`);
+    if (Configuration === "Debug") {
+        console.debug(`MONO_WASM: Debugger.Debug: ${message}`);
+    }
 }
 
 function _readSymbolMapFile(filename: string): void {
     try {
-        const res = Module.FS_readFile(filename, {flags: "r", encoding: "utf8"});
+        const res = Module.FS_readFile(filename, { flags: "r", encoding: "utf8" });
         res.split(/[\r\n]/).forEach((line: string) => {
-            const parts:string[] = line.split(/:/);
+            const parts: string[] = line.split(/:/);
             if (parts.length < 2)
                 return;
 
             parts[1] = parts.splice(1).join(":");
             wasm_func_map.set(Number(parts[0]), parts[1]);
         });
 
-        console.debug(`Loaded ${wasm_func_map.size} symbols`);
-    } catch (error:any) {
+        console.debug(`MONO_WASM: Loaded ${wasm_func_map.size} symbols`);
+    } catch (error: any) {
         if (error.errno == 44) // NOENT
-            console.debug(`Could not find symbols file ${filename}. Ignoring.`);
-        else
-            console.log(`Error loading symbol file ${filename}: ${JSON.stringify(error)}`);
+            if (Configuration === "Debug") {
+                console.debug(`MONO_WASM: Could not find symbols file ${filename}. Ignoring.`);
+            }
+            else
+                console.log(`MONO_WASM: Error loading symbol file ${filename}: ${JSON.stringify(error)}`);
```

Comment:
The else branch belongs to the outer if.

```suggestion
        if (error.errno == 44) { // NOENT
            if (Configuration === "Debug") 
                console.debug(`MONO_WASM: Could not find symbols file ${filename}. Ignoring.`);
        }
        else {
            console.log(`MONO_WASM: Error loading symbol file ${filename}: ${JSON.stringify(error)}`);
         }
```